### DISCUSSION TITLE
Official support for other language bindings in the SDK

### DISCUSSION THREAD

**Discussion by arun-msx** [microsoft/semantic-kernel#5060] [General]
Hello 👋 ,

I wanted to check if there are any plans in the pipeline for official support for languages other than C#, Python, Java. 
Especially, would be interested to know if Golang would be supported in the near future.

**Comment by stuartpa** [comment]
Any update on best way to get a semantic-kernel SDK for golang?  I see this github repo, is this the state of the art: https://github.com/mfmayer/gosk

**Comment by sophialagerkranspandey** [comment]
At this time, we don't have plans to create other languages. We'd be interested in hearing more about your use cases.

**Reply by xdave** [reply]
The `go` ecosystem has available to it a growing number of libraries for machine learning (including langchain, for example), and it's becoming an interesting space for these kinds of applications, due mostly to the type-safety, performance, and concurrency implications (in contrast to python, for example).
